src/olympia/reviewers/utils.py

```diff
@@ -800,7 +800,7 @@ def get_actions(self):
                 'add-on or versions.'
             ),
             'minimal': True,
-            'available': is_reviewer and has_unresolved_abuse_report_jobs,
+            'available': is_appropriate_reviewer and has_unresolved_abuse_report_jobs,
```

Comment:
Is this covered, most of the test_utils are just adding the string to a dict, so it's not very clear and based on the diff we are replacing one existing variable with another.

Reply:
mmm, you're right, it was a drive-by "fix", because I suddenly thought it should have a more restrictive condition.  I'll revert it - if it needs restricting it should be done with a separate issue & PR